test/functional/publications_controller_test.rb

```diff
@@ -217,7 +217,7 @@ def assert_featured(doc)
   end
 
   view_test "#index orders consultations by first_published_at date by default" do
-    consultations = 5.times.map {|i| create(:published_consultation, opening_at: (10 - i).days.ago) }
+    consultations = 5.times.map {|i| create(:published_consultation, first_published_at: (10 - i).days.ago) }
```

Comment:
Ordering consultations by published date rather than opening date is _possibly_ an undesirable side effect.